scene/3d/skeleton_3d.cpp

```diff
@@ -531,6 +540,50 @@ void Skeleton3D::set_bone_name(int p_bone, const String &p_name) {
 	version++;
 }
 
+Variant Skeleton3D::get_bone_meta(int p_bone, const StringName &p_key) const {
+	const int bone_size = bones.size();
+	ERR_FAIL_INDEX_V(p_bone, bone_size, "");
```

Comment:
```suggestion
	ERR_FAIL_INDEX_V(p_bone, bone_size, Variant());
```
Otherwise it'll return `Variant()` if there's a bone but it doesn't have a mega with the key, and `""` if there's no such bone